miner/worker.go

```diff
@@ -214,6 +217,7 @@ func newWorker(config *Config, chainConfig *params.ChainConfig, engine consensus
 	// Subscribe events for blockchain
 	worker.chainHeadSub = eth.BlockChain().SubscribeChainHeadEvent(worker.chainHeadCh)
```

Comment:
Can the `chainHeadSub` be removed?

Reply:
Yes, since its role now has been replaced/optimized by `newViewSub`